server/plugin/hooks.go

```diff
@@ -83,6 +83,11 @@ func (p *Plugin) MessageHasBeenPosted(c *plugin.Context, post *model.Post) {
 		}
 	}
 
+	if err = p.scheduleJob(mattermostUserID); err != nil {
+		p.API.LogError("Error while scheduling a job", "Error", err.Error())
```

Comment:
Instead of logging the error everywhere where you're calling `scheduleJob`, you can just log the error inside `scheduleJob`.